jujugui/static/gui/src/app/components/profile/bundle-list/bundle-list.js

```diff
@@ -100,10 +156,12 @@ class ProfileBundleList extends React.Component {
               title={name} />);
         });
         const path = window.jujulib.URL.fromLegacyString(bundle.id).path();
+        const version = `#${bundle.id.slice(-1)[0]}`;
```

Comment:
Please let's not manipulate charm URls manually. We are already using the URL lib in the line above. so this could be written as
```javascript
const url = window.jujulib.URL.fromLegacyString(bundle.id);
const path = url.path();
const revision = url.revision;
```